docs/reference/action_list.rst

```diff
@@ -801,3 +847,76 @@ Combine this with configuring a custom template and you'll have a list column fu
 
 .. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/4.x/reference/list_field_definition/
 .. _`here`: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type
+
+Advance Usage
+-------------
+
+Displaying sub entity properties
```

Comment:
In the 
```
**TODO**:
    * adding custom columns
    * targeting submodel fields using dot-separated notation
```
at the top of the doc, you can now remove 
```
targeting submodel fields using dot-separated notation
```

Reply:
And I would say the custom columns could be remove too, since we provide example with templates and there is "Displaying a non-model field" examples

Reply:
`Displaying a non-model field` Removed

Reply:
> In the
> 
> ```
> **TODO**:
>     * adding custom columns
>     * targeting submodel fields using dot-separated notation
> ```
> 
> at the top of the doc, you can now remove
> 
> ```
> targeting submodel fields using dot-separated notation
> ```

removed

Reply:
Can you remove also
```
**TODO**:
    * adding custom columns
```
To me, it's done. Custom columns are made with custom template. Or am I missing something ?